ospfd/ospf_asbr.c

```diff
@@ -664,16 +668,22 @@ struct ospf_lsa *ospf_originate_summary_lsa(struct ospf *ospf,
 
 		asel = (struct as_external_lsa *)lsa->data;
 		tag = (unsigned long)ntohl(asel->e[0].route_tag);
+		metric = (unsigned int)ntohl(GET_METRIC(asel->e[0].metric));
+		mtype = IS_EXTERNAL_METRIC(asel->e[0].tos);
```

Comment:
Should be `IS_EXTERNAL_METRIC(asel->e[0].tos) ? EXTERNAL_METRIC_TYPE_2 : EXTERNAL_METRIC_TYPE_1`.

Reply:
done